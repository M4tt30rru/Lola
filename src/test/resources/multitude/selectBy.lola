##Import "std.lola"
##Find ##Expression(exp) #st ##Expression(filter);
 ##replace (##(exp)).stream().filter(##(filter)).collect(Collectors.toList());

f() #st i -> i % 2 == 0;
