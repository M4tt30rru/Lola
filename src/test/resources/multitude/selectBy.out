(f()).stream().filter(i -> i % 2 == 0).collect(Collectors.toList());
